webapp/src/main.js

```diff
@@ -0,0 +1,51 @@
+import {ApolloClient} from 'apollo-client'
+import {ApolloLink} from 'apollo-link'
+import {HttpLink} from 'apollo-link-http'
+import {InMemoryCache} from 'apollo-cache-inmemory'
+import VueApollo from 'vue-apollo';
+import Vue from 'vue'
+import BootstrapVue from "bootstrap-vue"
+import App from './App.vue'
+import "bootstrap/dist/css/bootstrap.min.css"
+import "bootstrap-vue/dist/bootstrap-vue.css"
+import {setContext} from "apollo-link-context";
+import {AUTH_TOKEN} from './constants/settings.js'
+
+Vue.config.productionTip = false;
+
+const httpLink = new HttpLink({
+    uri: 'http://localhost:4000/graphql',
+});
+
+const authLink = setContext((_, {headers, ...context}) => {
+    const token = localStorage.getItem(AUTH_TOKEN);
+
+    return {
+        headers: {
+            ...headers,
+            ...(token ? {Authorization: `Bearer ${token}`} : {}),
```

Comment:
I'm very proud that at least one group implemented the bearer token in exercise 3, well done! :+1: 

You even implemented the frontend part, which was not required. Most groups tried pass the token via `args` to the resolver. `args` is a poor choice because it's available in only one resolver. The client would need to add the token in each nested resolver. With `context` you can decode it once and use the result in all resolvers (don't repeat yourself). `context` is the same for all resolvers, `args` is available only to one resolver.